.github/workflows/publish-docker-images.yaml

```diff
@@ -1,35 +1,44 @@
-name: cd
+name: Publish Docker Images
 
 on:
   push:
     tags:
       - "v*.*.*"
 
+permissions:
+  packages: write
+  contents: read
+
 jobs:
   docker-publish-mcp:
     name: docker-publish-mcp
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v4
+        uses: actions/checkout@v5
         with:
           submodules: recursive
       - name: Docker meta
         id: meta
         uses: docker/metadata-action@v5
         with:
           images: |
-            ghcr.io/timescale/tiger-slack/mcp
+            docker.io/timescale/tiger-slack-mcp
+            ghcr.io/timescale/tiger-slack-mcp
           tags: |
             type=raw,value=latest,enable=${{ github.ref == format('refs/heads/{0}', 'main') }}
             type=semver,pattern={{version}}
             type=semver,pattern={{major}}.{{minor}}
             type=semver,pattern={{major}}
-            type=sha
```

Comment:
Removed the sha publish since this isn't really useful for tagged publishing imo (where someone can already pin to a specific registry SHA if they want, vs git sha), only really useful for nightlies.